docs/user/communication.rst

```diff
@@ -228,80 +230,71 @@ in order to easily read the content by clicking the image.
-
 Features for Moderators
 -----------------------
 
 The following features are only available for moderators, not for students.
 
-Move Posts
-^^^^^^^^^^
-
-Tutors can change the context (lecture, exercise, course-wide topic) in the edit mode of the post.
-By changing the context, for example from a certain exercise to a course-wide topic, the post is automatically moved.
-In the example at hand, the post will not be shown on the according exercise page anymore, but rather only in the course-wide
-communication overview, associated with that certain course-wide topic.
-
-Pin Posts
-^^^^^^^^^
-
-By clicking the pushpin icon next to the reaction button of a post, a moderator can *pin* the post.
-As a consequence, the post is displayed at the top of any post list to receive higher attention.
+Pin Messages
+^^^^^^^^^^^^
 
-Archive Posts
-^^^^^^^^^^^^^
-
-As a complement to pinning, i.e., highlighting posts, a moderator can archive posts and thereby put them at the bottom of a post list.
-This can be achieved by clicking the folder icon next to the reaction button.
-Moderators should be aware that this reduces the visibility of the post.
+By clicking the pushpin icon next to the reaction button of a message, a moderator can *pin* the message.
+As a consequence, the message is highlighted to receive higher attention.
 
 Features for Instructors
 ------------------------
 
 The following feature is only available for instructors that act as moderators.
 
-Post Announcements
-^^^^^^^^^^^^^^^^^^
+Creating Course-Wide Channels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Instructors can create course-wide posts that serve as *Announcements*.
-They target every course participant and have higher relevance than normal posts.
-Announcements can be created in the course communication overview by selecting the topic ``Announcement``.
-As soon as the announcement is created, all participants, that did not actively refrain from being notified, will receive an email containing the announcement's content.
-Additionally, announcements visually differ from normal posts and are always displayed on top of the communication overview.
+When creating a lecture, exercise, or exam, the creator can decide for a channel name, as well. The name is automatically generated based on the lecture's/exercise's/exam's title but can be adapted if needed.
 
+|exercise-channel-creation|
+
+Message Announcements
+^^^^^^^^^^^^^^^^^^^^^
+
+Instructors can create course-wide messages that serve as *Announcements*.
+They target every course participant and have higher relevance than normal messages.
+Announcements can be created in the course messaging overview by selecting the ``Announcement`` channel.
+As soon as the announcement is created, all participants, that did not actively refrain from being notified, will receive an email containing the announcement's content.
```

Comment:
As soon as the announcement is created, all participants **who** did not actively refrain from being notified will receive an email containing the announcement's content. (remove commas around defining relative clause)